Products/ZenHub/metricpublisher/publisher.py

```diff
@@ -126,7 +126,7 @@ class RedisListPublisher(BasePublisher):
     """
 
     def __init__(self,
-                 host='localhost',
+                 host='rd1',
```

Comment:
I'm not sure how this ensures `rd1` is the host used.  See https://github.com/zenoss/zenoss-prodbin/blob/3919975aba0edf15ff3203a8546f1561b447a34b/Products/ZenHub/PBDaemon.py#L611-L612